x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/components/use_policies.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+
+import type { AgentPolicy } from '../../../../../../../../../common';
+
+import { SO_SEARCH_LIMIT } from '../../../../../../../../../common';
+import { useGetAgentPolicies } from '../../../../../../../../hooks';
+
+export const useAllNonManagedAgentPolicies = () => {
+  let existingAgentPolicies: AgentPolicy[] = [];
+  const { data: agentPoliciesData, error: err } = useGetAgentPolicies({
+    page: 1,
+    perPage: SO_SEARCH_LIMIT,
+    sortField: 'name',
+    sortOrder: 'asc',
+    full: false, // package_policies will always be empty
+    noAgentCount: true, // agentPolicy.agents will always be 0
+  });
+  if (err) {
+    // eslint-disable-next-line no-console
+    console.debug('Could not retrieve agent policies');
+  }
+  existingAgentPolicies = useMemo(
```

Comment:
could we search directly for non managed policies in the API call with a kuery like `not is_managed:* or is_managed:false` (not tested)?

Reply:
I tried that but it didn't work. I think `is_managed` is not indexed: https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/core/template-resources/src/main/resources/fleet-agents.json

Reply:
it should be indexed https://github.com/juliaElastic/kibana/blob/22c3f95868d6a5d804e9ed85b7575da6dd423b46/x-pack/plugins/fleet/server/saved_objects/index.ts#L282 but not a blocker

Reply:
You are right, I was looking at the wrong place.
The filter actually works, I tried before without the prefix `ingest-agent-policies.is_managed:false`